__tests__/Port.test.js

```diff
@@ -0,0 +1,32 @@
+/* global describes it expect */
+const Port = require('../src/Port');
+describe('Port', () => {
+    describe('monitors ships in and leaving port', () => {
+        let plymouth;
+        let ship;
+        let titanic;
+        let queenMary;
+        beforeEach(() => {
+            plymouth = new Port('Plymouth');
+            ship = jest.fn();
+            titanic = jest.fn();
+            queenMary = jest.fn();
+    });
+    it('can be instantiated', () => { 
```

Comment:
As above all tests should have a clear line above them to make it easier to read